src/main/kotlin/no/nav/familie/ef/sak/opplysninger/søknad/domain/Søknadsverdier.kt

```diff
@@ -52,6 +87,69 @@ fun SøknadsskjemaOvergangsstønad.tilSøknadsverdier() = Søknadsverdier(
     bosituasjon = this.bosituasjon,
     situasjon = this.situasjon,
     datoMottatt = this.datoMottatt,
-    søkerFra = this.søkerFra
-
+    søkerFra = this.søkerFra,
+    dokumentasjon = tilDokumentasjon(this)
 )
+
+fun tilDokumentasjon(søknadsskjema: SøknadsskjemaBarnetilsyn): DokumentasjonFraSøknadDto =
```

Comment:
Er de ulike dokumentene på ulike steder i de ulike søknadene, eller er det mulig å lage en metode som tar inn de ulike delene fra søknadene? Eks
```kotlin
fun tilDokumentasjon(søknadsskjema: SøknadsskjemaBarnetilsyn) = tilDokumentasjon(
søknadsskjema.aktivitet, søknadsskjema.sivilstand, ...
)

private fun tilDokumentasjon(aktivitet: Aktivitet, sivilstand: Sivilstand, ...)
```

Reply:
Kan se på det. 
For skolepenger og barnetilsyn så finnes ikke de samme "naturlige" objektene så dokumentasjon er lagt på et eget objekt. Men går an å sende inn dét, kanskje.